helper/userHelper.js

```diff
@@ -0,0 +1,53 @@
+const bcrypt = require("bcrypt");
+const User = require("../models/user");
+const generateToken = require("../utils/generateToken");
+const logger = require("../utils/logger");
+
+const registerHelper = async (req, res) => {
```

Comment:
Return from this function, each API should return some data
For registration, data entered by user.

Follow a same response format for each API.

Reply:
used name in response for registration and login